public/UI.js

```diff
@@ -42,7 +42,64 @@ class UIController {
                     </thead>
                     <tbody id="coding-status-body"></tbody>
                 </table>
-            <div id="last-update">Last updated: </div>
+                <div id="last-update">Last updated: </div>
+                <div class="accordion" id="accordionExample">
+                    <div class="card">
+                        <div id="headingOne">
+                            <h2 class="mb-0">
+                                <button onclick="window.scrollBy(0, 1000)" class="btn btn-brown" type="button" data-toggle="collapse" data-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
```

Comment:
Thanks for the improvements here. When you load the page the first time, the scroll down does not seem to work. You have to scroll up. It, however, works from then on.PTAL